docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md

```diff
@@ -0,0 +1,217 @@
+---
+id: flutter-integration-testing-android
+title: Flutter Android
+sidebar_label: Flutter Android
+description: Run your Flutter integration tests for Android
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Integration testing involves testing various modules of the software under development as a group to see whether they function together seamlessly.
```

Comment:
Unit tests and flutter widget tests validate individual classes, functions, or widgets. 
You will not be able to validate how individual pieces work together in whole or capture the performance of an app running on real devices. 
To perform these tasks you could use Flutters' integration test solution.

Reply:
done